### DISCUSSION TITLE
OPENAPI 3.0  with multiple examples not showing

### DISCUSSION THREAD

**Discussion by devjdeed** [GitbookIO/community#422] [4. Open API]
I noticed importing an openapi.json file with endpoints that have multiple example responses don't show on the UI unlike endpoints with single example response. Is it this a bug ?

**Comment by aikoii** [comment]
👋 @IbrahimSam96 Yes, at the moment, we only show the first example. We're making significant updates to our OpenAPI block, so hopefully you can expect improvements on this front very soon 🤞!
